package.json

```diff
@@ -27,14 +39,21 @@
     "eslint-plugin-promise": "^3.6.0",
     "eslint-plugin-react": "^7.5.1",
     "eslint-plugin-standard": "^3.0.1",
+    "ethers": "^2.1.3",
```

Comment:
we dont need this anymore, this is for arc-js

Reply:
Fixed at d4aa48d